contracts/rust/deployer/Cargo.toml

```diff
@@ -20,5 +21,8 @@ tokio = { workspace = true }
 tracing = "0.1.37"
 vbs = { workspace = true }
 
+[features]
+safesdk = []
```

Comment:
What is the rationale behind adding this feature? Is it just to avoid running a test by default?

Reply:
ah to run just tests related to the safe sdk

Reply:
well more so to run specifically the safesdk related tests

Reply:
Then I think we can put `safesdk` in the name of the test or put those into a separate file or test module. Adding a feature seems overkill for this.